modules/javafx.graphics/src/main/java/javafx/scene/robot/Robot.java

```diff
@@ -224,8 +224,9 @@ public void mouseClick(MouseButton... buttons) {
     }
 
     /**
-     * Scrolls the mouse wheel by the specified amount of wheel clicks. Positive
-     * {@code wheelAmt} scrolls up whereas negative scrolls down.
+     * Scrolls the mouse wheel by the specified amount of wheel clicks. A positive
+     * {@code wheelAmt} scrolls the wheel towards the user (up) whereas negative
+     * amounts scrolls the wheel away from the user (down).
```

Comment:
I see that the FX docs were backwards relative to AWT before this change, so "up" and "down" need to be switched to correct this (which is probably why I thought it was Windows that was wrong instead of Linux). It should be:

positive ... towards the user (down)
negative ... away from the user (up)